regularizepsf/builder.py

```diff
@@ -193,19 +139,25 @@ def build(self,
               images: list[str] | list[pathlib.Path] | np.ndarray | Generator,
               sep_mask: list[str] | list[pathlib.Path] | np.ndarray | Generator | None = None,
               hdu_choice: int | None = 0,
+              num_workers: int | None = None,
               interpolation_scale: int = 1,
               star_threshold: int = 3,
               average_method: str = 'median',
               percentile: float = 50,
               saturation_threshold: float = np.inf,
               image_mask: np.ndarray | None = None,
+              star_minimum: float = 0,
+              star_maximum: float = np.inf,
+              sqrt_compressed: bool = False,
               return_patches: bool = False) -> (ArrayPSF, dict):
```

Comment:
I know the other parameters aren't documented, but could you include the new ones you added in the docstring? We really should include all of them. You'll get an extra prize if you do haha.